### PR TITLE
Add differentiating example source for `Enum.ToString` formats

### LINKED ISSUE #35854: Show the difference between the formats G and F
There is a somewhat convoluted description of the enum string formats G and F but no different example. It would certainly help a lot to see how the same code leads to different outputs for both formats. Think of this as writing unit tests. What would you test to make sure both implementations can have different outputs?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c7b9c332-da26-ff62-8669-5697d5b9aa56
* Version Independent ID: 0049d705-a8ca-a4b5-c51d-b4f88bdf27e3
* Content: [Enumeration format strings](https://learn.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings)
* Content Source: [docs/standard/base-types/enumeration-format-strings.md](https://github.com/dotnet/docs/blob/main/docs/standard/base-types/enumeration-format-strings.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

- Update both C# and VB sample source to include another example, which helps to differentiate output.
- Edit pass / acrolinx.

Fixes #35854

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/standard/base-types/enumeration-format-strings.md](https://github.com/dotnet/docs/blob/f70e1b4d0d33d0c92dbccec9e6e16c50818e4056/docs/standard/base-types/enumeration-format-strings.md) | [Enumeration format strings](https://review.learn.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings?branch=pr-en-us-35873) |

### CODE REVIEW

File: samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs [hunk 1]
```diff
@@ -32,10 +33,11 @@ private static void ShowFSpecifier()
    {
       Console.WriteLine("F Specifier:");
       // <Snippet2>
-      Console.WriteLine(ConsoleColor.Blue.ToString("F"));       // Displays Blue
-      FileAttributes attributes = FileAttributes.Hidden |
-                                  FileAttributes.Archive;
-      Console.WriteLine(attributes.ToString("F"));   // Displays Hidden, Archive
+      Console.WriteLine(((DayOfWeek)7).ToString("F"));       // Monday, Saturday
```
Comment (tdykstra) [inline samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs:36]: Isn't this just Saturday? I don't see where Monday is coming from.

File: samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb [hunk 2]
```diff
@@ -23,43 +23,47 @@ Module modMain
...
         ' </Snippet1>
         Console.WriteLine()
     End Sub
 
     Private Sub ShowFSpecifier()
         Console.WriteLine("F Specifier:")
         ' <Snippet2>
-        Console.WriteLine(ConsoleColor.Blue.ToString("F"))         ' Displays Blue
+        Console.WriteLine((CType(7, DayOfWeek)).ToString("F"))    ' Displays 7
+        Console.WriteLine(ConsoleColor.Blue.ToString("F"))        ' Displays Monday, Saturday
```
Comment (tdykstra) [inline samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb:39]: "Displays Monday, Saturday" doesn't fit here.

**Review by tdykstra — APPROVED** [review]
LGTM aside from a couple of what look like copy/paste errors.

File: samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs [hunk 3]
```diff
@@ -32,10 +33,11 @@ private static void ShowFSpecifier()
...
+      Console.WriteLine(ConsoleColor.Blue.ToString("F"));    // Displays Blue
+
+      var attributes = FileAttributes.Hidden | FileAttributes.Archive;
+      Console.WriteLine(attributes.ToString("F"));           // Displays Hidden, Archive
```
Comment (IEvangelist) [inline samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs:40]:
```suggestion
      Console.WriteLine(attributes.ToString("F"));           // Hidden, Archive
```

File: samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs [hunk 4]
```diff
@@ -44,10 +46,11 @@ private static void ShowDSpecifier()
...
       // <Snippet3>
-      Console.WriteLine(ConsoleColor.Cyan.ToString("D"));         // Displays 11
-      FileAttributes attributes = FileAttributes.Hidden |
-                                  FileAttributes.Archive;
-      Console.WriteLine(attributes.ToString("D"));                // Displays 34
+      Console.WriteLine(((DayOfWeek)7).ToString("D"));       // 7
+      Console.WriteLine(ConsoleColor.Cyan.ToString("D"));    // Displays 11
+
+      var attributes = FileAttributes.Hidden | FileAttributes.Archive;
+      Console.WriteLine(attributes.ToString("D"));           // Displays 34
```
Comment (IEvangelist) [inline samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs:53]:
```suggestion
      Console.WriteLine(ConsoleColor.Cyan.ToString("D"));    // 11

      var attributes = FileAttributes.Hidden | FileAttributes.Archive;
      Console.WriteLine(attributes.ToString("D"));           // 34
```

File: samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs [hunk 5]
```diff
@@ -56,10 +59,11 @@ private static void ShowXSpecifier()
...
       // <Snippet4>
-      Console.WriteLine(ConsoleColor.Cyan.ToString("X"));   // Displays 0000000B
-      FileAttributes attributes = FileAttributes.Hidden |
-                                  FileAttributes.Archive;
-      Console.WriteLine(attributes.ToString("X"));          // Displays 00000022
+      Console.WriteLine(((DayOfWeek)7).ToString("X"));       // 00000007
+      Console.WriteLine(ConsoleColor.Cyan.ToString("X"));    // Displays 0000000B
+
+      var attributes = FileAttributes.Hidden | FileAttributes.Archive;
+      Console.WriteLine(attributes.ToString("X"));           // Displays 00000022
```
Comment (IEvangelist) [inline samples/snippets/csharp/VS_Snippets_CLR/Formatting.Enum/cs/enum1.cs:66]:
```suggestion
      Console.WriteLine(ConsoleColor.Cyan.ToString("X"));    // 0000000B

      var attributes = FileAttributes.Hidden | FileAttributes.Archive;
      Console.WriteLine(attributes.ToString("X"));           // 00000022
```

File: samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb [hunk 6]
```diff
@@ -23,43 +23,47 @@ Module modMain
     Private Sub ShowGSpecifier()
         Console.WriteLine("G Specifier:")
         ' <Snippet1>
-        Console.WriteLine(ConsoleColor.Red.ToString("G"))           ' Displays Red
+        Console.WriteLine((CType(7, DayOfWeek)).ToString("G"))    ' Displays 7
+        Console.WriteLine(ConsoleColor.Red.ToString("G"))         ' Displays Red
         Dim attributes As FileAttributes = FileAttributes.Hidden Or _
                                            FileAttributes.Archive
-        Console.WriteLine(attributes.ToString("G"))     ' Displays Hidden, Archive                               
+        Console.WriteLine(attributes.ToString("G"))               ' Displays Hidden, Archive
```
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb:30]:
```suggestion
        Console.WriteLine(attributes.ToString("G"))               ' Hidden, Archive
```

File: samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb [hunk 7]
```diff
@@ -23,43 +23,47 @@ Module modMain
     Private Sub ShowGSpecifier()
         Console.WriteLine("G Specifier:")
         ' <Snippet1>
-        Console.WriteLine(ConsoleColor.Red.ToString("G"))           ' Displays Red
+        Console.WriteLine((CType(7, DayOfWeek)).ToString("G"))    ' Displays 7
+        Console.WriteLine(ConsoleColor.Red.ToString("G"))         ' Displays Red
```
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb:27]:
```suggestion
        Console.WriteLine((CType(7, DayOfWeek)).ToString("G"))    ' 7
        Console.WriteLine(ConsoleColor.Red.ToString("G"))         ' Red
```

File: samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb (hunk 2 above)
Comment (IEvangelist) [inline samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb:39]:
```suggestion
        Console.WriteLine(ConsoleColor.Blue.ToString("F"))        ' Blue
```

Comment (IEvangelist) [reply samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Enum/vb/enum1.vb]: Yes, you're correct. I've updated this...

**Pull request merged**